Cargo.toml

```diff
@@ -34,6 +34,7 @@ hostname = "0.3.1"
 indicatif = { version = "0.17", features = ["tokio"], optional = true }
 libc = "0.2.139"
 multibase = { version = "0.9.1", optional = true }
+netlink-packet-route = "0.15.0"
```

Comment:
can you make these new dependencies linux only as well?

Reply:
good call, done